resolver.go

```diff
@@ -73,16 +70,15 @@ func (r *Resolver) Run(tmpl Templater, w Watcherer) (ResolveEvent, error) {
 	output, err := gcViews(func() ([]byte, error) {
 		return tmpl.Execute(w.Recaller(tmpl))
 	})
-
-	switch { // specific to general
-	case err == ErrNoNewValues:
-		return ResolveEvent{NoChange: true}, nil
-	case !w.Complete(tmpl):
-		return ResolveEvent{missing: true}, nil
-	case err == nil:
+	switch {
+	case err == ErrNoNewValues || err == nil:
 	default:
 		return ResolveEvent{}, err
 	}
 
-	return ResolveEvent{Complete: true, Contents: output}, nil
+	return ResolveEvent{
```

Comment:
How would you imagine this change for `Complete` affecting the Buffer check L50-52?

Reply:
Hmm... while I think that *was* OK, I'm thinking if the explicit watcher registering could impact the buffer code where it checks for initialization. Going to see if I can add a test to check that (to that PR, not this one).

Reply:
Just wanted to follow up on this thread to make sure I understand! @eikenb, if I understand correctly, you might have recently pushed a new change regarding buffering. I don't remember the code prior. Would you mind describing the change? Everything makes sense, just want to make sure I understand the buffer stuff!

Reply:
Ah! Sorry for the confusion. Looks like your buffer changes are in the other PR you have open. Ignore my previous comment. Thank you!

Reply:
PR #61 fixes the issue with the Buffer() that I mention. Otherwise I think this is OK for now. There is one possible issue... that the ResolveEvent that is returned during buffering don't have the content which would otherwise always be available. 

IMO this is more a feature than a bug, but should probably be addressed. I see 2 possibilities...
1) add a method to the template to let the resolver grab the cached content
2) have it return an error when buffering

I'm leaning towards option 2 but will open a new ticket for this as it really isn't directly related to either this nor the cached template PR.

Reply:
Makes sense! Thanks for clarifying and opening the issue to discuss the how content should be returned for Buffer()